test/with_api_v2/acceptance.go

```diff
@@ -252,16 +256,45 @@ type Alertmanager struct {
 	errc chan<- error
 }
 
+// AlertmanagerCluster represents a group of Alertmanager instances
+// acting as a cluster.
+type AlertmanagerCluster struct {
+	ams []*Alertmanager
+}
+
+// Start the Alertmanager cluster and wait until it is ready to receive.
+func (amc *AlertmanagerCluster) Start() {
+	var peerFlags []string
+	for _, am := range amc.ams {
+		peerFlags = append(peerFlags, "--cluster.peer="+am.clusterAddr)
+	}
+
+	for _, am := range amc.ams {
+		am.Start(peerFlags)
+	}
+
+	for _, am := range amc.ams {
+		err := am.WaitForCluster(len(amc.ams))
+		if err != nil {
+			// TODO: Properly handle this error
```

Comment:
is this common? we probably should fail the test if for some reason the AM can't start